lib/senders/email.js

```diff
@@ -135,7 +135,7 @@ module.exports = function (log, config) {
     this.initiatePasswordResetUrl = config.initiatePasswordResetUrl
     this.iosUrl = config.iosUrl
     this.iosAdjustUrl = config.iosAdjustUrl
-    this.mailer = sender || nodemailer.createTransport(options)
+    this.mailer = require('./rust_mailer')
```

Comment:
I don't think we want to do a hard switchover at this point. Ideally we want a setting in config that can be used to flick it on or off. Maybe we even want that setting to be augmented by a regex that allows us to e.g. only send specific email addresses via the new service, and maybe even to send some email addresses via the SES provider and some via the Sendgrid provider. So it could be a map of provider names (keys) and regular expressions (values). Possibly there's other aspects we want to make configurable too, haven't thought of any yet but I'll keep stewing on it.